source/css/sass/patterns/molecules/teaser.scss

```diff
@@ -0,0 +1,25 @@
+@import "../../mixins/spacing";
+@import "../../mixins/typography";
+@import "../../mixins/_decorations";
+
+.teaser-list__list {
+  @include padding(0, inline-start);
+}
+
+.teaser {
+
+  .teaser-list__item:not(:last-child) & {
+    @include margin($baselinegrid-space-small, block-end);
```

Comment:
This is why my preference would be for the divider to be on the list item itself, so the `block-spacing` can be reduced by the divider height like https://github.com/libero/pattern-library/blob/31c20bdf6ef5dc3ae27cb8348c1775c03aebb1e4/source/css/sass/patterns/organisms/content-header.scss#L12-L13

So something like:

```scss
.teaser-list__item {
  &:not(:last-of-type) {
    @include block-spacing($end: $baselinegrid-space-small - $grid-divider_size);
    @include divider(block-end);
  }
}
```

Reply:
This was what I was referencing the other day. I did the initial work on the list item and when reviewed I moved it to teaser. Seems as if we are doing unnecessary work.